media-libs/libfpx/libfpx-1.3.1.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="FlashPIX Open Source Toolkit"
+DESCRIPTION="The FlashPIX Open Source Toolkit has library routines which handles FlashPIX images \
+and is based on source code obtained from the Digital Imaging Group Inc. (currently known as International \
+Imaging Industry Association or i3a)."
+HOMEPAGE="http://www.i3a.org"
+
+LICENSE="Flashpix"
+COPYRIGHT="1999 Digital Imaging Group, Inc."
+REVISION="6"
+
+SOURCE_URI="http://www.imagemagick.org/download/delegates/libfpx-1.3.1-6.tar.xz"
+CHECKSUM_SHA256="8ea8df5454f1a96c8a49446b7a0d3f7374ce87c501f90e64f4d6ea2d735b0c94"
+
+ARCHITECTURES="x86 x86_64 x86_gcc2 arm"
```

Comment:
Since you only tested it with x86_gcc2 other architectures should be marked as untested like this: ?x86